dune/xt/functions/test/indicator_function.tpl

```diff
@@ -126,20 +128,18 @@ TEST_F(IndicatorFunction_from_{{GRIDNAME}}_to_{{r}}_times_{{rC}}, global_evaluat
     DomainType lower_left(ll);
     for (auto ur : {1., 0.75, 0.5, 0.25}) {
```

Comment:
Just FYI, you can use our ```XT::Common::valueRange(start,stop,delta)``` in these kinds of situations.

Reply:
Great thanks